pkg/registry/core/pod/strategy_test.go

```diff
@@ -1393,45 +1397,6 @@ func TestNodeInclusionPolicyEnablementInUpdating(t *testing.T) {
 		*createdPod.Spec.TopologySpreadConstraints[0].NodeTaintsPolicy != honor {
 		t.Error("NodeInclusionPolicy created with unexpected result")
 	}
-
```

Comment:
I think we shouldn't remove this part; rather should preserve this part with `SetFeatureGateEmulationVersionDuringTest`?